scalatags/shared/src/main/scala/scalatags/Escaping.scala

```diff
@@ -62,8 +64,7 @@ object Escaping {
         case '\n' => s.append('\n')
         case '\r' => s.append('\r')
         case '\t' => s.append('\t')
-        case c if c < ' ' =>
-        case c => s.append(c)
+        case c => if (c >= ' ') s.append(c)
```

Comment:
Why not the following?

```suggestion
        case c if c >= ' ' => s.append(c)
```

Reply:
Well, technically your version is a non-exhaustive match. It seems scala doesn't warn once you have a guarded case like this, though.

Reply:
Fair point about the non-exhaustive match. I did not think about that.